Code/GraphMol/Descriptors/AUTOCORR3D.cpp

```diff
@@ -0,0 +1,240 @@
+//
```

Comment:
This copyright statement is wrong.
Please include the correct year and your own name (if you can do that) or nothing at all and I'll add something later.